### PR TITLE
engine: 26.0.0 release notes

### PR DESCRIPTION
**Pull request by vvoland**
Engine release notes (draft, but no changes are expected).

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: hugo.yaml [hunk 1]
```diff
@@ -94,12 +94,12 @@ params:
...
 
-  latest_engine_api_version: "1.44"
-  docker_ce_version: "25.0.4"
+  latest_engine_api_version: "1.45"
+  docker_ce_version: "26.0.0"
   compose_version: "v2.25.0"
   compose_file_v3: "3.8"
   compose_file_v2: "2.4"
-  buildkit_version: "0.12.3"
+  buildkit_version: "0.13.0"
```
Comment (sam-thibault) [inline hugo.yaml:102]: This should be `0.13.1`, right?

**Review by sam-thibault — CHANGES_REQUESTED** [review]

File: hugo.yaml (hunk 1 above)
Comment (vvoland) [reply hugo.yaml]: Yes, fixed, thanks!

File: content/engine/release-notes/26.0.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+- Remove deprecated aliases from the api/types package. These types were deprecated in v25.0.0, which provided temporary aliases. [moby/moby#47148](https://github.com/moby/moby/pull/47148)
+  These aliases are now removed: `types.Info`, `types.Commit`, `types.PluginsInfo`, `types.NetworkAddressPool`, `types.Runtime`, `types.SecurityOpt`, `types.KeyValue`, `types.DecodeSecurityOptions`, `types.CheckpointCreateOptions`, `types.CheckpointListOptions`, `types.CheckpointDeleteOptions`, `types.Checkpoint`, `types.ImageDeleteResponseItem`, `types.ImageSummary`, `types.ImageMetadata`, `types.ServiceUpdateResponse`, `types.ServiceCreateResponse`, `types.ResizeOptions`, `types.ContainerAttachOptions`, `types.ContainerCommitOptions`, `types.ContainerRemoveOptions`, `types.ContainerStartOptions`, `types.ContainerListOptions`, `types.ContainerLogsOptions`
+- cli/command/container: remove deprecated `NewStartOptions()` [docker/cli#4811](https://github.com/docker/cli/pull/4811)
+- cli/command: remove deprecated `DockerCliOption`, `InitializeOpt`[docker/cli#4810](https://github.com/docker/cli/pull/4810)
```
Comment (dvdksn) [inline content/engine/release-notes/26.0.md:123]: We don't normally include changes to Go modules. Do we want to keep these? @thaJeztah

**Review by neersighted — CHANGES_REQUESTED** [review]
I've carried this at https://github.com/docker/docs/pull/19658 since we ended up with some differences in the final upstream release notes; I'll also try to bring the unresolved discussions over.

**Pull request closed without merging**